### PR TITLE
feb release

### PR DESCRIPTION
**Pull request by tatanithin**
feb release with below changes
1:Disable cvp till cvp is up
2:lab menu into ui
3:switch status with realtime update and fix functionality
4:password as popup

### CODE REVIEW

File: nested-labvm/atd-docker/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   uptime:
     container_name: atd-uptime
-    image: us.gcr.io/atd-testdrivetraining-dev/atddocker_uptime:1.0.0
+    image: us.gcr.io/testdrive-traininglabs-alpha-b/atddocker_uptime:1.0.0
```
Comment (ruhansharief22) [inline nested-labvm/atd-docker/docker-compose.yml]: change back to atd dev images

File: nested-labvm/atd-docker/uilanding/src/uilanding.py [hunk 2]
```diff
@@ -25,6 +26,21 @@
 
 ArBASE_PATH = '/opt/modules/'
 MODULE_FILE = ArBASE_PATH + 'modules.yaml'
+MENU_BASE_PATH = '/opt/menus/'
+# Open yaml for the default yaml and read what file to lookup for default menu
+default_menu_file = open(MENU_BASE_PATH+'default.yaml')
```
Comment (ruhansharief22) [inline nested-labvm/atd-docker/uilanding/src/uilanding.py:31]: Needs exception handling. 
Open yaml file using "with" keyword and keep it in the try except block

File: nested-labvm/atd-docker/uilanding/src/uilanding.py [hunk 3]
```diff
@@ -282,6 +305,52 @@ def pS(mtype):
...
+        login_container = docker_conn.containers.get('atd-login')
+        container_output=login_container.exec_run(f'sudo lab_status.py')
+        log_file = open('log.txt','w')
+        log_file.write(str(container_output.output.decode("utf-8")))
+        log_file.close()
+        with open("log.txt", "r") as txt_file:
+            response =  txt_file.readlines()
+        print(response)
+        self.write({
+            'response':response
```
Comment (ruhansharief22) [inline nested-labvm/atd-docker/uilanding/src/uilanding.py:338]: also account for the scenario:
if the container exits with an error, then the error is logged in the log.txt file. in such case, some default error message should be sent as response.

File: nested-labvm/atd-docker/uilanding/src/uilanding.py [hunk 4]
```diff
@@ -282,6 +305,52 @@ def pS(mtype):
...
+        login_container = docker_conn.containers.get('atd-login')
+        container_output=login_container.exec_run(f'python3 /usr/local/bin/callConfigTopo.py  {DEFAULT_MENU_FILE_VALUE} {selected_lab_option}')
+        print(container_output)
+        log_file = open('log.txt','w')
+        log_file.write(str(container_output.output.decode("utf-8")))
+        log_file.close()
+        with open("log.txt", "r") as txt_file:
+            response =  txt_file.readlines()
+        self.write({
+            'response':response
```
Comment (ruhansharief22) [inline nested-labvm/atd-docker/uilanding/src/uilanding.py:322]: also account for the scenario:
if the container exits with an error, then the error is logged in the log.txt file. in such case, some default error message should be sent as response.

File: topologies/base_topo.yml [hunk 5]
```diff
@@ -629,7 +629,7 @@ topologies:
             resources:
               cpu: 40
               ram: 80
-            branch: nested/2021.3
+            branch:   lab-option-switch-status-nested-release
```
Comment (ruhansharief22) [inline topologies/base_topo.yml]: revert this change

**Review by ruhansharief22 — COMMENTED** [review]
consider the review comment and address if valid

File: nested-labvm/atd-docker/docker-compose.yml (hunk 1 above)
Comment (tatanithin) [reply nested-labvm/atd-docker/docker-compose.yml]: will be modified

File: nested-labvm/atd-docker/uilanding/src/uilanding.py (hunk 2 above)
Comment (tatanithin) [reply nested-labvm/atd-docker/uilanding/src/uilanding.py:31]: default.yaml file is mandatory and part of build process. it will be available in all topologies

File: topologies/base_topo.yml (hunk 5 above)
Comment (tatanithin) [reply topologies/base_topo.yml]: this will be changed

File: nested-labvm/atd-docker/uilanding/src/uilanding.py (hunk 3 above)
Comment (tatanithin) [reply nested-labvm/atd-docker/uilanding/src/uilanding.py:338]: That is fine, 99% atd-login container will not fail because it is must required container.
switch details are extracted from files which are created as part of build process.
 lab_status.py have try/catch functionality, switch will be down if connectivity is failed. 
collectively there are very  less chances to occur error

File: nested-labvm/atd-docker/uilanding/src/uilanding.py (hunk 4 above)
Comment (tatanithin) [reply nested-labvm/atd-docker/uilanding/src/uilanding.py:322]: That is fine, 99% atd-login container will not fail because it is must required container.
here the same error message will be displayed from log.txt to UI.

**Review by ruhansharief22 — APPROVED** [review]
